### DISCUSSION TITLE
Edges for instancedMesh

### DISCUSSION THREAD

**Discussion by kayden1940** [pmndrs/drei#1188] [Q&A]
It seems using Edges on instancedMesh would not work out of the box.

I then have stumble upon a post below about using edgesgeometry instancedmesh.
https://discourse.threejs.org/t/use-edgesgeometry-in-an-instancedmesh/16723/4

Not sure how to implement it in r3f.

**Comment by drcmda** [comment]
you can do what the post you linked says you should. just use plain THREE.EdgesGeometry and that offset shader extension whatever that does. in react anything that is imperative goes into a useLayoutEffect or useEffect, you can access the instance with useRef, there's really no difference from there on.
